### ISSUE TITLE
Infinite loop in compiler

### ISSUE THREAD

**Issue by ixje** [CityOfZion/neo3-boa#895]
**Describe the bug**
I had a user ask me why he couldn't import some functions. I tried to create a minimal contract to see what was going on and the compiler just went into an infinite loop. Here the last bit before I was able to terminate it

```
ERROR: 594:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 596:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 599:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 601:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 603:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 605:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 607:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 609:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 611:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 613:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 616:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>
ERROR: 618:4 - Internal compiler error. AttributeError: 'NoneType' object has no attribute 'class_variables'	 </home/erik/Documents/code/neo3-boa/boa3/neo/vm/opcode/Opcode.py>

```

**How to Reproduce**
Put this into a file and try to compile it

```python
from boa3.neo import to_script_hash, to_hex_str, from_hex_str
```
**Expected behavior**
No infinite loop

**Platform:**
 - OS: ubuntu 20.04
 - Version: neo3-boa v0.11.2
 - Python version: 3.9.9

**Comment by meevee98** [comment]
We're going to verify the infinity loop, but just for noting: those methods are not available for smart contracts.
The compiler only accepts the package `boa3.builtin` for importing.
All boa built-in methods available can be found [in the docs](https://dojo.coz.io/neo3/boa/package-reference.html).

**Comment by ixje** [comment]
The imports I used above is what the user tried todo. To quote him
>  I have NigVWQwT8Mc4ZkxEDsaEtuvL8hZYQFHr5A address, and want to get public key in neo3-boa in format 

> any idea of how to make 
bytes.fromhex("19650b64eb614c295053cf044b74d8cf9dab3c41040e3c88f181dcf583f37ba8")
in neo3-boa library?
[9:52 AM]
from boa3.neo import to_script_hash, to_hex_str, from_hex_str
tried this one, but import fails on compilation (edited)

I did a quick scan of the documentation but I don't think it explicitly mentions that `boa3.builtin` is the only allowed package to import. Or did I miss the statement? Perhaps we can black list all other packages under `boa3` and warn the user that `boa3.builtin` is the only valid package to import. The alternative is to restructure and put everything else under `boa3.internal` as a way to signal to the user not to use it. That's might be a very big undertaking.
